test/expected/build/library.n/dest/resources/library/n/designtime/Button.designtime.js

```diff
@@ -3,4 +3,4 @@
  * (c) Copyright 2009-xxx SAP SE or an SAP affiliate company.
```

Comment:
All these comments `UI development toolkit for HTML5 (OpenUI5)` should only be present if the file is copied or directly derived from the OpenUI5 repository.

For example in this case the code in this file has no relation to the code OpenUI5 repository anymore. Hence the comment should be removed. Please also review the other new and changed files in this PR for this!

Reply:
done

Reply:
I was not aware that this comes from the configured copyright string and is part of the test. Changing "UI development toolkit for HTML5 (OpenUI5)" to "UI5 Tooling" would have been ok as well in this case. Now we have fewer tests for multiline copyright strings, oh well...

Reply:
tests for `library.e` still have the multiline comment